epos_hardware/install_udev_rules

```diff
@@ -4,11 +4,13 @@ echo ""
 echo "This script copies a udev rule for ftd2xx devices"
 echo ""
 
-sudo cp `rospack find epos_hardware`/90-ftd2xx.rules /etc/udev/rules.d
+sudo cp `rospack find epos_hardware`/rules/90-ftd2xx.rules /etc/udev/rules.d
+sudo cp `rospack find epos_hardware`/rules/99-epos4.rules /etc/udev/rules.d
+sudo cp `rospack find epos_hardware`/rules/99-ftdi.rules /etc/udev/rules.d
```

Comment:
these are both `99`, do you want to differentiate the numbers to make the order explicit?